node_modules/oae-authentication/lib/util.js

```diff
@@ -24,11 +27,40 @@ var TenantsUtil = require('oae-tenants/lib/util');
 /**
  * Setup the necessary authentication middleware
  *
- * @param  {express} server     An express server
+ * @param  {Object}     config  The OAE system configuration
```

Comment:
We usually use `An object containing the full system configuration (i.e., \`config.js\`)` for the full config object